app/locale/en/messages.po

```diff
@@ -0,0 +1,70 @@
+msgid ""
+msgstr ""
+"POT-Creation-Date: 2021-12-10 22:51+0100\n"
+"Mime-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: @lingui/cli\n"
+"Language: en\n"
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Plural-Forms: \n"
+
+#: components/views/Home/index.tsx:325
+msgid "footer.blog"
+msgstr "blog"
+
+#: components/views/Home/index.tsx:326
+msgid "footer.community"
+msgstr "community"
+
+#: components/views/Home/index.tsx:324
+msgid "footer.docs"
+msgstr "docs"
+
+#: components/views/Home/index.tsx:323
+msgid "footer.home"
+msgstr "home"
+
+#: components/views/Home/index.tsx:235
+msgid "header.connectedto"
+msgstr "You are connected to"
```

Comment:
I could probably answer this question by reading the docs but,
does this mean in order to change a string, we need to change it in the component, but also here in the messages file?

Or is this auto-generated?

Reply:
You need to change it only in the `messages.po` files. The strings in the component are only default values.